corehq/apps/domain/views/internal.py

```diff
@@ -212,19 +212,26 @@ class FlagsAndPrivilegesView(BaseAdminProjectSettingsView):
     def dispatch(self, request, *args, **kwargs):
         return super(FlagsAndPrivilegesView, self).dispatch(request, *args, **kwargs)
 
-    @memoized
-    def enabled_flags(self):
-        def _sort_key(toggle_enabled_tuple):
-            toggle, domain_enabled, user_enabled = toggle_enabled_tuple
-            return (not domain_enabled,
-                    not user_enabled,
-                    toggles.ALL_TAGS.index(toggle.tag),
-                    toggle.label)
-        unsorted_toggles = [(
-            toggle,
-            toggle.enabled(self.domain, namespace=toggles.NAMESPACE_DOMAIN),
-            toggle.enabled(self.request.couch_user.username, namespace=toggles.NAMESPACE_USER)
-        ) for toggle in toggles.all_toggles()]
+    def _get_toggles(self):
```

Comment:
Was removing `@memoized` deliberate?

Reply:
Yup - the method was only referenced once